src/main/java/frc/robot/Constants.java

```diff
@@ -333,6 +333,8 @@ public static final class ShooterConstants {
     public static final double kIdleSpeed = 15.0;
     public static final double kShootingSpeed = 25.0;
 
+    public static final double kSpinFactor = 0.0;
```

Comment:
This is a constant *term* rather than a *factor*. A name like `kSpinSpeed` would be okay, but `kSpinFactor` misleads.